API/BookSharing/BookSharing.Infrastructure/SeedData/BooksSeedData.cs

```diff
@@ -10,9 +10,9 @@ public static async Task Seed(BookSharingDbContext dbContext)
         {
             dbContext.Books.AddRange(new[]
             {
-                new Book(ISBN: "9788324626625", Title: "Wzorce projektowe. Elementy oprogramowania obiektowego wielokrotnego użytku", Author:"Erich Gamma", Year:"1994-10-21", Description:"Book about design patterns" ),
-                new Book(ISBN: "9788328305250", Title: "Domain-Driven Design. Zapanuj nad złożonym systemem informatycznym", Author:"Eric Evans", Year:"2015-07-06", Description:"Book about Domain-Driven Design"),
-                new Book(ISBN: "9788308068144", Title: "Sapiens. Od zwierząt do bogów", Author:"Yuval Noah Harari", Year:"2018-11-14", Description:"Book about how man came into being"),
+                new Book(ISBN: "9788324626625", Title: "Wzorce projektowe. Elementy oprogramowania obiektowego wielokrotnego użytku","1"),
```

Comment:
jezeli wszedzie w klasie nadawane sa prefixy zmiennej np ISBN:, to we wszystkich polach powinny byc dodane, ta 1 to w sumie nie wiadomo co